dangerzone/gui/main_window.py

```diff
@@ -480,77 +492,78 @@ def check_state(self) -> None:
         error: Optional[str] = None
 
         try:
-            if isinstance(  # Sanity check
-                self.dangerzone.isolation_provider, Container
-            ):
-                container_runtime = self.dangerzone.isolation_provider.get_runtime()
-                runtime_name = self.dangerzone.isolation_provider.get_runtime_name()
+            self.dangerzone.isolation_provider.is_runtime_available()
         except NoContainerTechException as e:
             log.error(str(e))
             state = "not_installed"
-
+        except NotAvailableContainerTechException as e:
+            log.error(str(e))
+            state = "not_running"
+            error = e.error
```

Comment:
I would also add the following except clause:

```diff
+        except Exception as e:
+            log.error(str(e))
+            state = "not_running"
+            error = format_exception(e)
```

While toying with the PR (renaming the `docker.exe` file to something else), I managed to get an `OSError` exception here. It's best to catch everything, just to be on the safe side.

Reply:
Good catch! I've updated the code in 8da6d30af68979dc43239aa749071d967fa00441